cmd/config.go

```diff
@@ -97,6 +98,7 @@ func newConfigCommand(config *settings.Config) *cobra.Command {
 	processCommand.Flags().StringP("org-slug", "o", "", "organization slug (for example: github/example-org), used when a config depends on private orbs belonging to that org")
 	processCommand.Flags().String("org-id", "", "organization id used when a config depends on private orbs belonging to that org")
 	processCommand.Flags().StringP("pipeline-parameters", "", "", "YAML/JSON map of pipeline parameters, accepts either YAML/JSON directly or file path (for example: my-params.yml)")
+	processCommand.Flags().StringP("circleci-api-host", "", "", "the api-host you want to use for config processing and validation")
```

Comment:
Dumb question - what does this line do?

Reply:
This allows us to set the circleci-api-host explicitly as a global flag. I've added it so that we can add a cucumber test that explicitly hits the k9 instance.